modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -1378,17 +1378,13 @@ public Set<NodeWithAttributes> logicalTopology() {
     }
 
     private void registerCatalogEventListenersOnStartManagerBusy() {
-        catalogManager.listen(ZONE_CREATE, (parameters, exception) -> inBusyLock(busyLock, () -> {
-            assert exception == null : parameters;
-
+        catalogManager.listen(ZONE_CREATE, parameters -> inBusyLock(busyLock, () -> {
```

Comment:
Same thing: we can use an explicit parameter type and make this code even shorter

Reply:
fix it